src/pybind/mgr/dashboard/frontend/src/openattic-theme.scss

```diff
@@ -1,1144 +0,0 @@
-/*
```

Comment:
Have you removed this file?

Reply:
I have removed this file and copied all important and global classes and styles to styles.scss file.